src/bindings/python/PyLogCameraTransform.cpp

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#include "PyTransform.h"
+
+namespace OCIO_NAMESPACE
+{
+
+void bindPyLogCameraTransform(py::module & m)
+{
+    LogCameraTransformRcPtr DEFAULT = LogCameraTransform::Create();
+
+    std::array<double, 3> DEFAULT_LOG_SIDE_SLOPE;
+    DEFAULT->getLogSideSlopeValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LOG_SIDE_SLOPE.data()));
+
+    std::array<double, 3> DEFAULT_LOG_SIDE_OFFSET;
+    DEFAULT->getLogSideOffsetValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LOG_SIDE_OFFSET.data()));
+
+    std::array<double, 3> DEFAULT_LIN_SIDE_SLOPE;
+    DEFAULT->getLinSideSlopeValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LIN_SIDE_SLOPE.data()));
+
+    std::array<double, 3> DEFAULT_LIN_SIDE_OFFSET;
+    DEFAULT->getLinSideOffsetValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LIN_SIDE_OFFSET.data()));
+
+    std::array<double, 3> DEFAULT_LIN_SIDE_BREAK;
+    DEFAULT->getLinSideBreakValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LIN_SIDE_BREAK.data()));
+
+    std::array<double, 3> DEFAULT_LINEAR_SLOPE;
+    DEFAULT->getLinearSlopeValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LINEAR_SLOPE.data()));
+
+    py::class_<LogCameraTransform, 
+               LogCameraTransformRcPtr /* holder */, 
+               Transform /* base */>(m, "LogCameraTransform")
+        .def(py::init(&LogCameraTransform::Create))
+        .def(py::init([](const std::array<double, 3> & logSideSlope,
+                         const std::array<double, 3> & logSideOffset,
+                         const std::array<double, 3> & linSideSlope,
+                         const std::array<double, 3> & linSideOffset,
+                         const std::array<double, 3> & linSideBreak,
+                         const std::array<double, 3> & linearSlope,
```

Comment:
Typically, the linearSlope param will not be used.  The param vec in the object does not contain a default for this case, it is simply a shorter vec.  (Actually, I'm surprised the initializer on line 29 doesn't throw.)

I think it would be better to remove linearSlope from the constructor so that it is not initialized.  People will use the setter if they need to use that param (which is not the common scenario).

Reply:
Thanks for the info @doug-walker. I'll remove it from the constructor.

In testing this particular transform further I'm noticing an issue that may prevent this convenience constructor for being beneficial with default parameter values. There is no default value for `LinSideBreak` defined by the class, but the defaults for the other four params result in a `validate()` failure unless `LinSideBreak` is set. The params can be set to valid values, but this may necessitate not utilizing defaults, but only supporting this constructor if all values are explicitly set. This is the exception thrown by `validate()` with the default transform state:
```
LogCameraTransform validation failed: LinSideBreak has to be defined.
```
Is that the intended behavior? How would you suggest we provide a convience contructor given the intended usage?

Reply:
Yes, linSideBreak needs to be set and there is no obvious default as there are for the other ones.  So this argument for the convenience constructor cannot be optional.

Reply:
Changed.